client-mvp-04/src/store/configureStore.js

```diff
@@ -0,0 +1,15 @@
+  
+import { createStore, combineReducers } from 'redux';
+import authReducer from '../reducers/authReducer';
+
+export default () => {
+    // Store creation
+    const store = createStore(
+        combineReducers({
+            auth: authReducer,
+        }), /* preloadedState, */
+        window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
+    );
+    
+    return store;
+}
```

Comment:
I think it will be great to create a separate file to combine all reducers in one place:
`reducers/index.js`

```
import authReducer from './authReducer';
import { combineReducers } from 'redux';

const allReducers = combineReducers({
    auth: authReducer
})

export default allReducers;
```

And update the store with code:
```
import { createStore } from 'redux';
import allReducers from '../reducers';

export default () => {
    return createStore(
        allReducers,
        window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
    );
}
```